SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -136,8 +136,10 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 
 		if (EntityId != SpatialConstants::INVALID_ENTITY_ID)
 		{
-			// If the entity already exists, make sure we have spatial authority before we replicate with Offloading, because we pretend to have local authority
-			if (USpatialStatics::IsSpatialOffloadingEnabled(GetWorld()) && !bCreatingNewEntity && !NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::POSITION_COMPONENT_ID))
+			// If the entity already exists, make sure we have spatial authority before we replicate with multi-worker, because we pretend to have local authority.
+			if (USpatialStatics::IsSpatialMultiWorkerEnabled(GetWorld())
```

Comment:
If you replace the IsSpatialMultiWorkerEnabled with (NetDriver->LoadBalanceStrategy->GetMinimumRequiredWorkers() > 1) it will include the override check automagically.